maven-resolver-provider/src/main/java/org/apache/maven/repository/internal/DefaultVersionRangeResolver.java

```diff
@@ -195,6 +199,18 @@ private Map<String, ArtifactRepository> getVersions( RepositorySystemSession ses
         return versionIndex;
     }
 
+    private boolean isEnabled( RemoteRepository remoteRepository, String version )
+    {
+        if ( remoteRepository == null )
+        {
+            return true;
+        }
+
+        boolean snapshot = version != null && version.endsWith( SNAPSHOT );
```

Comment:
What about timestamped snapshots?

Reply:
Hi @cstamas ,

Thank you for looking at this. I am not I understand this comment. The check is similar to the code in the DefaultVersionResolver (where it determines whether a local file is a SNAPSHOT version by checking that it ends with 'SNAPSHOT' (https://github.com/apache/maven/blob/master/maven-resolver-provider/src/main/java/org/apache/maven/repository/internal/DefaultVersionResolver.java#L208).

Can you explain a bit more what you mean by "timestamped snapshots" ?